build-system/tasks/check-analytics-vendors-list.js

```
@@ -71,7 +71,10 @@ async function checkAnalyticsVendorsList() {
     const name = nameMatch
       .split(/[,\s]+/)
       .shift()
-      .replace(/[`"']/g, '');
+      ?.replace(/[`"']/g, '');
```

Use `!` and let it fail fast.

---

Can't! This isn't a JS file

---

Arg. Keeping it as-is, lmk if you'd like a followup